### PR TITLE
Add RBi for stdlib/rexml

### PR DESCRIPTION
**Pull request by Morriar**
No signature.

### CODE REVIEW

**Comment by jez** [comment]
https://go/builds/bui_HJZAu5hCvHXwLm

**Comment by jez** [comment]
When I ran this on Stripe's CI, this caused a small handful of errors which isn't usually a problem except that I can't see a quick (5 minute) solution to fixing them. So we might have to take a while longer to get back to you before we can merge this one.

**Comment by jez** [comment]
Alright after further checking, this isn't quite as hard to change on Stripe's end but it will take a little bit.

**Comment by Morriar** [comment]
Any news on this one?

**Comment by jez** [comment]
I can look at this today

File: rbi/stdlib/rexml.rbi [hunk 1]
```diff
@@ -0,0 +1,3413 @@
...
+  #
+  #
+  # ```
+  # XPath.each( node ) { |el| ... }
+  # XPath.each( node, '/*[@attr='v']' ) { |el| ... }
+  # XPath.each( node, 'ancestor::x' ) { |el| ... }
+  # XPath.each( node, '/book/publisher/text()=$publisher', {}, {"publisher"=>"O'Reilly"}) \
+  #   {|el| ... }
+  # ```
+  def self.each(element, path = _, namespaces = _, variables = _, &block); end
```
Comment (jez) [inline rbi/stdlib/rexml.rbi:3348]:
```suggestion
  def self.each(element, path = _, namespaces = _, variables = _, options = _, &block); end
```

File: rbi/stdlib/rexml.rbi [hunk 2]
```diff
@@ -0,0 +1,3413 @@
...
+  #     injection attacks or to automatically handle escaping string values.
+  #
+  #
+  # ```
+  # XPath.first( node )
+  # XPath.first( doc, "//b"} )
+  # XPath.first( node, "a/x:b", { "x"=>"http://doofus" } )
+  # XPath.first( node, '/book/publisher/text()=$publisher', {}, {"publisher"=>"O'Reilly"})
+  # ```
+  def self.first(element, path = _, namespaces = _, variables = _); end
```
Comment (jez) [inline rbi/stdlib/rexml.rbi:3372]:
```suggestion
  def self.first(element, path = _, namespaces = _, variables = _, options = _, &block); end
```

File: rbi/stdlib/rexml.rbi [hunk 3]
```diff
@@ -0,0 +1,3413 @@
...
+
+  # Returns an array of nodes matching a given
+  # [`XPath`](https://docs.ruby-lang.org/en/2.6.0/REXML/XPath.html).
+  def self.match(element, path = _, namespaces = _, variables = _); end
```
Comment (jez) [inline rbi/stdlib/rexml.rbi:3376]:
```suggestion
  def self.each(element, path = _, namespaces = _, variables = _, options = _, &block); end
```

**Pull request merged**